packages/docs/docusaurus.config.js

```diff
@@ -55,7 +55,8 @@ const config = {
         logo: {
           alt: 'DXOS',
           src: 'img/dxos-horizontal.svg',
-          srcDark: 'img/dxos-horizontal-white.svg'
+          srcDark: 'img/dxos-horizontal-white.svg',
+          target: "_blank"
```

Comment:
suggestion: Add a comment here on why this is required so it doesn't get removed unexpectedly

Reply:
Good catch!